tests/app/file_utils/test_file_utils.py

```diff
@@ -21,8 +21,8 @@
 
 def test_get_notification_references_from_s3():
     filenames = [
-        '2017-12-06/NOTIFY.ABC0000000000000.D.2.C.C.20171206184702.PDF',
-        '2017-12-06/NOTIFY.DEF0000000000000.D.2.C.C.20171206184710.PDF'
+        '2017-12-06/NOTIFY.ABC0000000000000.SUFFIX1.PDF',
```

Comment:
Optional, I think a bonus would be to keep both.

One test case for `suffix1` to make sure the code can handle generic suffixes and one for what we really expect the filename to look like

Reply:
Sure, that's fair. I was originally aiming to reduce the maintenance effort if we ever changed the name again, but that's a far fetched goal. It's certainly possible the extra segments could change the result.

I've changed the test to use a mixture of both.